### ISSUE TITLE
Bulk selections are lost if you click to the next page of results

### ISSUE THREAD

**Issue by julesies** [samvera-labs/nurax-pre2023#141]
### Descriptive summary
If you go to your works, and you want to add a bunch of works to a collection, and these works are not on the same "page" -- you expect to "keep" selected check boxes as you move through pages of works. However, if you navigate from page to page, and then go back, these selections are gone. 

1. Log in as a user with many works
2. Go to Dashboard: Works: Your Works
3. Select a few items on the next page, by checking the box next to the works
4. Click on page 3 of the search result, select something,
5. Go back to page one, see nothing selected

**Comment by elrayle** [comment]
This is not related to collection extensions.  The same thing happens at master.

@jcoyne @mjgiarlo @vantuyls Do you know if this is a regression or has it always been this way?

**Comment by jcoyne** [comment]
I don't think this has ever worked in the way that @julesies expects it to work. Well, back in very early Sufia it was session based, but then it had trouble when you had two windows open.

**Comment by vantuyls** [comment]
i don't believe this is functionality we tested for the 2.0 release.

**Comment by julesies** [comment]
@vantuyls right, wasn't meaning it was a new bug, just something of value to address because of CE use case makes the need for this more clear.

**Comment by vantuyls** [comment]
i'd like us to consider fixing this for the 2.1 release. discuss?

**Comment by elrayle** [comment]
@vantuyls My hesitation to putting this on the list of MustFix issues is that it never worked the way it is described in the issue statement.  So it represents new code rather than a bug fix.  I would prefer to put this on the list for 2.1.1.

I'm not sure how much work would be involved.  And we have limited bug fix resources for this release.
